### PR TITLE
👌 Improve plots with labels and units

### PR DESCRIPTION
**Pull request by s-weigand**
This PR addresses some of the paper review remarks and improves the labels of plots and adds units.

### Change summary

- [👌 Improved plot labeling after review](https://github.com/glotaran/pyglotaran-release-paper-supplementary-information/commit/f69284f0b970fa6ccc530f12880fd989cf4aad31)
- [👌 Change 'spectral' to 'Wavelength' in subplot title](https://github.com/glotaran/pyglotaran-release-paper-supplementary-information/commit/47b01e61b53e35d19428e09581f4fb982adaacad)

### Checklist

- [x] ✔️ Passing the tests (mandatory for all PR's)

### CODE REVIEW

**Review by jsnel — COMMENTED** [review]
Please rebase and see the diff in CI generated plots.

File: 4TT/sequential_doas_4TT.ipynb [hunk 1]
```diff
@@ -330,7 +336,9 @@
     "for ax in axes.flatten():\n",
     "    ax.axhline(0, color=\"k\", linewidth=1)\n",
     "    ax.set_xlabel(\"Time (ps)\")\n",
-    "    ax.set_ylabel(\"\")"
+    "    ax.set_ylabel(\"\")\n",
+    "    ax_title = ax.get_title()\n",
+    "    ax.set_title(rf\"{ax_title.replace('spectral', 'Wavelength')}$\\,$nm\")"
```
Comment (jsnel) [inline 4TT/sequential_doas_4TT.ipynb:341]: If replacing title spectral with wavelength, perhaps also change x-axis from time to Time (ps).  

I wouldn't mind the fontsize a little bigger.

**Review by jsnel — COMMENTED** [review]
This adds correct axis titles for those figures that are relevant to the paper, but not everywhere. Notably, the Data overview plots still have their 'raw' labels, as well as the plot_overview. That's fine as long as the figures are not directly used in the publication, else I would also dress them up.

I would modify the 'plot_traces' plots to label time as Time (ps).

File: 4TT/sequential_doas_4TT.ipynb (hunk 1 above)
Comment (jsnel) [reply 4TT/sequential_doas_4TT.ipynb:341]: This is one example, but there are multiple cases in the multiple notebooks.

Comment (s-weigand) [reply 4TT/sequential_doas_4TT.ipynb:341]: We are not using any `plot_overview`plot in the publication, but we should add a more convenient way to change axis labels and titles on bulk in the extras at some point.

**Review by jsnel — APPROVED** [review]
Looks good now.

**Pull request merged**